src/message/no_std_io.rs

```diff
@@ -0,0 +1,208 @@
+//! https://docs.rs/not-io/0.1.0-alpha/not_io/
```

Comment:
Why do we need that?

Reply:
This basically is a work around for `std::io::Error` and `std::io::Result<>`. If you're asking about the link, then I forgot to remove it :P

Reply:
Well, why we need io::Error? Can you just remove it?